src/module.d.ts

```diff
@@ -308,14 +308,22 @@ export namespace plugins {
     interface PositionSettings {
         // The height of the table
         tableHeight?: number|string;
+
         // The width of the table
         tableWidth?: number|string;
+
         // The minimum row height
         rowHeight?: number|string;
+
+        // TODO: Unused?
```

Comment:
I believe all the "Unused" notes are from `propTypes` that don't seem to have any usage. For a few of the `propTypes` (e.g. `keyColumn`) the correct name seems to have changed.